site/docs/master/release.md

```diff
@@ -1,61 +1,77 @@
-  * Update the minimum and maximum Kubernetes API versions that Sonobuoy supports.
-    Edit `pkg/buildinfo/version.go` and update the `MinimumKubeVersion` to be 2 minor version below the new Kubernetes release version and update the `MaximumKubeVersion` to support future point releases.
-    For example, for the Kubernetes 1.15.0 release, the `MinimumKubeVersion` would become `1.13.0` and the `MaximumKubeVersion` would become `1.15.99`.
-1. Commit and open/merge a pull request with these changes.
-1. Create an annotated tag for the commit once the changes are merged:
+  * Update the `kind-config.yaml` file with the new image version [here](https://github.com/vmware-tanzu/sonobuoy/blob/master/kind-config.yaml).
+
+### Setup e2e image registry for new version
+Add the new list of E2E test images for the targed version.
+
+Within `pkg/image/`, copy the latest `v1.x.go` file to a file which corresponds to the new Kubernetes release number.
+
+For example, if the new Sonobuoy release corresponds to Kubernetes `v1.20`, copy the `v1.19.go` file to `v1.20.go`.
+
+```
+cd pkg/image
+cp v1.{19,20}.go
+```
+         
+This file will contain a function to return the list of test images via a `map[int]Config` value.
+1. Update the name of the function accordingly. For example, if the file is for the v1.20 release, 
+         ensure the function name is `v1_20`.
+1. Copy all `map` entries stored in variable `configs` from upstream Kubernetes file [`kubernetes/test/utils/image/mafnifest`](https://github.com/kubernetes/kubernetes/blob/e19964183377d0ec2052d1f1fa930c4d7575bd50/test/utils/image/manifest.go#L202-L252). Remove the previous map entries and replace them wi the The copied entries.
+2. Update the `GetImageConfigs` function within `pkg/image/manifest.go` and add a new `case` statement to the minor version check which will be the minor version of the new Kubernetes release.  In this new case, call the newly created function (e.g. `r.v1_20()`).
+3. Add the new default image registry configuration by updating the function `GetDefaultImageRegistries` within `pkg/image/manifest.go` to return the default image registries for the new version.
+    * Add a new case to the minor version check which will correspond the minor version of the new Kubernetes release.
+    * Follow previous case statements (i.e. `case 19:`), return a new `RegistryList` object which includes only the registry keys used within the registry config for that version.
+    Some registries are not applicable to include in this object as they are there to test specific image pull behavior such as pulling from a private or non-existent registry. This object should only include registries that can be successfully pulled from. The other registries are not used within the end-to-end tests.
+
+### Updating the versioned docs
```

Comment:
This should be moved up a level in headings (h2) as it should be done for every release, not just new K8s releases.

Reply:
👍